pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -167,6 +167,7 @@ stdenv.mkDerivation (finalAttrs: {
     "-Dtracker3=${lib.boolToString trackerSupport}"
     "-Dbroadway_backend=${lib.boolToString broadwaySupport}"
     "-Dx11_backend=${lib.boolToString x11Support}"
+    "-Dwayland_backend=${lib.boolToString waylandSupport}"
```

Comment:
I think this is fine but I really don't know

Reply:
Without this parameter a build error occurs.